src/EditorFeatures/Core/Shared/Utilities/WorkspaceThreadingService.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.Shared.Utilities
+{
+    [Export(typeof(IWorkspaceThreadingService))]
+    [Shared]
```

Comment:
What is the style here? I typically keep the `Shared` attribute on the same line

Reply:
I always put one attribute in one set of brackets